### PR TITLE
Fixed issue in Zoho that caused contacts to inherit values from previous contacts

### PR DESCRIPTION
**Pull request by alanhartless**
[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | y

[//]: # ( Required: )
#### Description:

There was a bug where if a contact had a blank value, it assumed the value of the previous contact that didn't. 

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Setup the zoho plugin and map required fields
2. Create several contacts in Mautic the last one without a last name
3. Sync with Zoho `php app/console m:i:s -i Zoho -a "1 day"`
4. Look in Zoho for the contact without the last name in Mautic
5. It'll have the last name of the previous contact

#### Steps to test this PR:
1. repeat the above; this time the contact without a last name should fail sync because last name is required; it should not inherit the previous contact's last name
2. Code review and run the new unit tests

### CODE REVIEW

File: plugins/MauticCrmBundle/Tests/Api/Zoho/Xml/WriterTest.php [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+namespace MauticPlugin\MauticCrmBundle\Tests\Api\Zoho\Xml;
+
+use MauticPlugin\MauticCrmBundle\Api\Zoho\Xml\Writer;
+
+class WriterTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @testdox Test that a single row is generated correctly with numerical index
+     *
+     * @covers  \Writer::row()
```
Comment (Maxell92) [inline plugins/MauticCrmBundle/Tests/Api/Zoho/Xml/WriterTest.php]: There has to be a full path `\MauticPlugin\MauticCrmBundle\Api\Zoho\Xml\Writer`. CS fixer adds a slash here :(

**Comment by mqueme** [comment]
This worked

**Comment by mqueme** [comment]
@escopecz that has been fixed and is in current staging.

**Comment by mqueme** [comment]
tested Leads, Contacts, Accounts. works

**Pull request merged**